src/w3c/headers.js

```diff
@@ -433,11 +433,11 @@ export function run(conf) {
     });
   if (conf.bugTracker) {
     if (conf.bugTracker.new && conf.bugTracker.open) {
-      conf.bugTrackerHTML = `<a href='${conf.bugTracker.new}'>${conf.l10n.file_a_bug}</a> ${conf.l10n.open_parens}<a href='${conf.bugTracker.open}'>${conf.l10n.open_bugs}</a>${conf.l10n.close_parens}`;
+      conf.bugTrackerHTML = hyperHTML`<a href='${conf.bugTracker.new}'>${conf.l10n.file_a_bug}</a> ${conf.l10n.open_parens}<a href='${conf.bugTracker.open}'>${conf.l10n.open_bugs}</a>${conf.l10n.close_parens}`;
     } else if (conf.bugTracker.open) {
-      conf.bugTrackerHTML = `<a href='${conf.bugTracker.open}'>open bugs</a>`;
+      conf.bugTrackerHTML = hyperHTML`<a href='${conf.bugTracker.open}'>open bugs</a>`;
     } else if (conf.bugTracker.new) {
-      conf.bugTrackerHTML = `<a href='${conf.bugTracker.new}'>file a bug</a>`;
+      conf.bugTrackerHTML = hyperHTML`<a href='${conf.bugTracker.new}'>file a bug</a>`;
```

Comment:
Same here, please replace `${[conf.bugTrackerHTML]}` into `${conf.bugTrackerHTML}` in two `src/w3c/templates/(cgbg-)headers.js` files.